ArchUnitNET/Domain/PlantUml/PlantUmlDiagram.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("ArchUnitNETTests")]
+namespace ArchUnitNET.Domain.PlantUml
+{
+    internal class PlantUmlDiagram
```

Comment:
should all those domain classes be internal? or could some be accessible to be used without the fluent api? Like most other domain classes

Reply:
My intention was to encapsulate the whole parser code. This code should not by part of the libraries public API as it is an implementation detail.

Reply:
ok, sounds reasonable. However, this somehow implies that this feature won't be able with the non-fluent API. which is fine, but maybe there is a nice way to enable it later.